### ISSUE TITLE
[Other] Collaboration proposal

### ISSUE THREAD

**Issue by XargonWan** [rommapp/romm#30]
Greetings,

I'm Xargon, the founder of RetroDECK and before of that developer on 351ELEC/AmberELEC.

With another developer I recently started a project named RetroMan (https://github.com/RetroMan-project).
Our project just started and we noticed your RomM project and we are interested in it as I found it somehow similar to ours.

RetroMan stands for "Retro Manager", it amis to be a fully pluginable core for managing user's rom libraries.
RetroMan, is focusing on recognizing and syncing the roms on different devices, let's say that you have a miyoo mini and a game boy flahs cart, you can decide which roms (and saves/states) to keep synced in the various devices by having one (or more) main library.
So there will be a "server mode" and a "client" mode to allow the user to use it to catalogue their roms and to manage their physical devices.

Basically every device got two templates: the device template and the physical device file that they are linked together to instruct RetroMan about the device's features, supported systems, folder locations, etc.
These files are yaml that they can always be updated.

Then it can generate gameslists.xml that suits the devices frontend, such as EmulationStation or Onion for example.

It seems like we have similar goals, at least for the server side, so I would like to discuss the evenutality to unify the projects and join our forces to do a single better software and avoid fragmentation.

Feel free to check our page and ask any questions :)

Kindly,
Xargon

**Comment by crispybegs** [comment]
this looks interesting. will there be a docker image?

**Comment by crispybegs** [comment]
> Yes of course :)
> 
> [RetroMan-project/core#6](https://github.com/RetroMan-project/core/issues/6)

thanks. i don't quite understand whether you mean there''s an existing docker option or that one is coming, but I'd love something that can keep something like RomM and my RG353VS in sync somehow.

**Comment by XargonWan** [comment]
> > Yes of course :)
> > [RetroMan-project/core#6](https://github.com/RetroMan-project/core/issues/6)
> 
> thanks. i don't quite understand whether you mean there''s an existing docker option or that one is coming, but I'd love something that can keep something like RomM and my RG353VS in sync somehow.

Yes, sorry, it's a planned feature. 
Actually our scope is even doing the part of Romm, that's why I opened this issue.

**Comment by zurdi15** [comment]
Greetings!

Frist of all, I am glad that you are interested in RomM. I checked RetroMan and it indeed seems like we have similar goals (to be honest, you initial scope is way bigger than mine). 

I started RomM as a personal solution for my personal needs, but eventually I decided to publish it to the public to allow anyone who wants to use it (or even contribute) do it. 

As this is for now a hobbie in terms of time disposal and effort, and even I agree with you that would be better a single better software than two fragmented options, sadly (at least for now) I am going to reject your offer because right now my personal situation doesn't allow me to commit to other people.

In any case this could change in the future, and even if that is not the case and as you said at least in the server side our golas are pretty similar, eventually I could adapt RomM to RetroMan core/API or whichever interface are you going to develop.

I know this is not the answer you maybe were expecting but as I said, right now I can't colaborate with other people because I can't compromise myself with other people and be a bad co-worker.

I will keep an eye in RetroMan and sorry if my english is not the best.

Thank you for your offer and your hard work with RetroMan

Sincerely,
Zurdi
